### PR TITLE
Task 6 completed

### PR DESCRIPTION
**Pull request by SimonForshaw**
Task 6 completed and ready for review.

### CODE REVIEW

File: src/components/Reviews/CommentCard.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import * as api from "../../../Api";
+import dayjs from "dayjs";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+const CommentCard = () => {
+  const [comments, setComments] = useState([]);
+  const { review_id } = useParams();
+  console.log(comments);
```
Comment (EddTheDuck) [inline src/components/Reviews/CommentCard.jsx:9]: Please remember to remove console.logs before submitting.

File: src/components/Reviews/CommentCard.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  useEffect(() => {
+    api.fetchCommentsById(review_id).then((data) => {
+      setComments(data.comment);
+    });
+  }, []);
+  return (
+    <div>
+      {comments.map((comment) => {
+        console.log(comment);
```
Comment (EddTheDuck) [inline src/components/Reviews/CommentCard.jsx:18]: Another log here

**Review by EddTheDuck — APPROVED** [review]
Looks good, just remember to remove console.logs

File: src/components/Reviews/CommentCard.jsx (hunk 2 above)
Comment (SimonForshaw) [reply src/components/Reviews/CommentCard.jsx:18]: will do, Thanks!

**Pull request merged**